### PR TITLE
Styling webpage

### PR DESCRIPTION
**Pull request by Lucas-Masaba**
Add CSS styles to awesome books

### CODE REVIEW

File: JS/ui.js [hunk 1]
```diff
@@ -50,11 +69,24 @@ class UI {
...
     const newBookToAppend = document.createElement('li');
 
+    //   <div>
+    //   <h3>"${book.title}" </h3>
+    //   <span> by ${book.author}</span>
+    // </div>
+    // <button
+    //   data-id="${book.id}"
+    //   type="button">Remove
+    // </button>
```
Comment (cyonii) [inline JS/ui.js]: - [x] Please do not leave commented pieces of code in your codebase as they make your project look messy and unprofessional. You can use git history if you need to review a previous version of your project.

File: index.html [hunk 2]
```diff
@@ -6,47 +6,37 @@
...
+      <header>
+        <h1 class="awesome_books">Awesome books</h1>
+      </header>
+      <ul id="books">
+
       </ul>
       <script type="module">
         UI.displayBooks()
-      </script>      
+      </script>
```
Comment (cyonii) [inline index.html]: - [x]  Please keep `script` tags before the closing `</body>` tag so the downloading and executing it doesn't block the browser to parse the `HTML` in the document. It also separates the `HTML` and `JavaScript` code and helps to manage the codebase better.

**Review by cyonii — CHANGES_REQUESTED** [review]
Hi @Lucas-Masaba & @gabrie-lhilarion, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review to see the changes you need to make._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: JS/ui.js (hunk 1 above)
Comment (Lucas-Masaba) [reply JS/ui.js]: Thank you for your observation. I have just removed the comment

File: index.html (hunk 2 above)
Comment (Lucas-Masaba) [reply index.html]: Thank you for that keen observation. We have managed to place it right

**Review by EricMbouwe — APPROVED** [review]
### Hi @Lucas-Masaba, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**